io.catenax.quality_message_content/1.0.0/QualityMessageContent.ttl

```diff
@@ -0,0 +1,141 @@
+#######################################################################
```

Comment:
will this be a shared aspect? if so, it should be moved to the namespace io.catenax.shared...